### PR TITLE
Update Form Recognizer blog for beta 3

### PR DESCRIPTION
**Pull request by samvaity**
Update Form recognizer blog post for reflecting preview/beta 3 changes for the client library.

### CODE REVIEW

File: _posts/2020-05-19-formrecognizer.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Introducing Form Recognizer client library: AI-powered document extraction"
 layout: post
-date: 2020-05-19
+date: 2020-06-10
```
Comment (iscai-msft) [inline _posts/2020-05-19-formrecognizer.md:4]: Isn't the date for release going to be 2020 06-19

Comment (samvaity) [reply _posts/2020-05-19-formrecognizer.md:4]: Not sure, once I get a signoff from our team will add Adrian to confirm this!

File: _posts/2020-05-19-formrecognizer.md [hunk 2]
```diff
@@ -109,19 +118,36 @@ The below example shows how you can get the bounding information for the field `
...
-      // Prints the total price of the item
-      System.out.printf("Total Price: %s, confidence: %.2f%n", receiptItem.getTotalPrice().getFieldValue(), receiptItem.getTotalPrice().getConfidence());
-    });
+    if (receiptItemsField.getFieldValue().getType() == FieldValueType.LIST) {
+      List<FormField> receiptItems = receiptItemsField.getFieldValue().asList();
+      receiptItems.forEach(receiptItem -> {
+        if (receiptItem.getFieldValue().getType() == FieldValueType.MAP) {
+            receiptItem.getFieldValue().asMap().forEach((key, formField) -> {
+              // Prints the name of the item
+              if (key.equals("Name")) {
```
Comment (mssfang) [inline _posts/2020-05-19-formrecognizer.md:130]: nit: if ("Name".equals(key))

File: _posts/2020-05-19-formrecognizer.md [hunk 3]
```diff
@@ -266,7 +292,45 @@ Next, let's use the above created custom model to analyze our custom form.
...
+Firstly, let's obtain authorization to copy into the target resource by calling the `getCopyAuthorization()` operation against the target resource client.
+```java
+// Instantiate the target client where we want to copy the custom model to.
+FormTrainingClient targetClient = new FormTrainingClientBuilder()
+    .credential(new AzureKeyCredential("{key}"))
+    .endpoint("https://{target_endpoint}.cognitiveservices.azure.com/")
+    .buildClient();
+
+// Get authorization to copy the model to target resource
+CopyAuthorization copyAuthorization = targetClient.getCopyAuthorization("targetResourceId", "targetResourceRegion");
```
Comment (mssfang) [inline _posts/2020-05-19-formrecognizer.md:311]: nit: want to use "{target_resource_id"} for the user's input? also apply same for `targetResourceRegion` and "copy-model-id"

File: _posts/2020-05-19-formrecognizer.md (hunk 1 above)
Comment (adrianhall) [reply _posts/2020-05-19-formrecognizer.md:4]: Do not change the date - you are doing an update to the blog, not a new blog.

File: _posts/2020-05-19-formrecognizer.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ Let's see some code on how the app might want to use this library. Although, you
 
 ## Recognize expense fields from receipt
 
-The Form Recognizer prebuilt receipt API includes a pre-trained model for reading English sales receipts (e.g. receipts from restaurants or gas stations). This model has already been trained to extract expense related key information such as the time and date of the transaction, merchant information, taxes, total cost, individual line items, and more.
+The Form Recognizer prebuilt receipt API includes a pre-trained model for reading English sales receipts (e.g. receipts from restaurants or gas stations). This model has already been trained to extract expense related key information such as the time and date of the transaction, merchant information, taxes, total cost, individual line items, and [more]( https://westus2.dev.cognitive.microsoft.com/docs/services/form-recognizer-api-v2-preview/operations/GetAnalyzeReceiptResult).
```
Comment (adrianhall) [inline _posts/2020-05-19-formrecognizer.md:20]: Why add a link when one wasn't present in the initial blog?  This is just screaming for yet another update later on.  We expect blogs to go out of date, but broken links need to be fixed.  Please revert.

File: _posts/2020-05-19-formrecognizer.md [hunk 5]
```diff
@@ -266,7 +286,45 @@ Next, let's use the above created custom model to analyze our custom form.
...
 
-The client library differentiates between training with labels and without labels with the presence of boolean parameter `useLabelFile`. If `useLabelFile = false` the `beginTraining` API performs model training without labels.
+The client library differentiates between training with labels and without labels with the presence of boolean parameter `useTrainingLabels`. If `useTrainingLabels = false` the `beginTraining` API performs model training without labels.
+
+### What's new in Form Recognizer?
+#### Copy custom model
+The Form Recognizer client library now supports copying a custom model from one form recognizer resource to another. You can now copy your custom trained models between regions and subscriptions using the new Copy Custom Model feature.
```
Comment (adrianhall) [inline _posts/2020-05-19-formrecognizer.md:293]: This whole section doesn't make sense in an old blog.  It looks like you added this back in May.  This isn't a "new blog" - it's an update.  Keep it to making the code work.  Remove this section.

File: _posts/2020-05-19-formrecognizer.md (hunk 4 above)
Comment (samvaity) [reply _posts/2020-05-19-formrecognizer.md:20]: This was something we changed in P3 and the link should help guide the user for understanding the code and its usage.

File: _posts/2020-05-19-formrecognizer.md (hunk 5 above)
Comment (samvaity) [reply _posts/2020-05-19-formrecognizer.md:293]: So we don't show `New features` in the update that were added as a part of P3?

Comment (samvaity) [reply _posts/2020-05-19-formrecognizer.md]: Removed, updated.

**Pull request merged**